docs/building-a-form/common-definitions.md

```diff
@@ -171,16 +160,16 @@ A function that takes a full name `schema` and sets its required field list to `
 - `schema`: Yes
 - Function or object: Function
 
-##### Person ID
+### Person ID
```

Comment:
This is pretty specific to Vets.gov, so perhaps we should consider removing it from our library?

Reply:
Do you mean update the description of Person ID to remove the VA file number reference, or remove the definition altogether? If the latter, I'll wait to remove the docs until the definition is removed.

Reply:
Ah I was thinking removing the definition altogether, which I agree we can do as a separate PR, so nothing to do here for now.